openapi.yaml

```
@@ -16985,6 +17032,346 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X POST -d '{
+                "label": "PG_Miami_failover",
+                "region": "us-mia",
+                "affinity_type": "anti-affinity:local",
+                "is_strict": true
+              }' \
+              https://api.linode.com/v4/placement/groups
+      - lang: CLI
+        source: >
+          linode-cli placement create \
+            --label PG_Miami_failover \
+            --region us-mia \
+            --affinity_type "anti-affinity:local" \
+            --is_strict true
+  /placement/groups/{groupId}:
+    x-linode-cli-command: placement
+    parameters:
+    - name: groupId
+      in: path
+      description: ID of the placement group to look up. Run the [Placement Groups List](#placement-groups-list) operation and store the `id` for the applicable placement group.
+      required: true
+      schema:
+        type: integer
+    get:
+      x-linode-grant: read-only
+      tags:
+      - Placement Groups
+      summary: Placement Group View
+      description: |
+        View a specific placement group by ID.
+      operationId: getPlacementGroup
+      x-linode-cli-action: view
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_only
+      responses:
+        '200':
+          description: Returns a single placement group object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
+      x-code-samples:
+        - lang: Shell
+          source: >
+            curl -H "Authorization: Bearer $TOKEN" \
+                -X GET \
+                https://api.linode.com/v4/placement/groups/528
+        - lang: CLI
+          source: >
+            linode-cli placement view 528
+    put:
+      x-linode-grant: read-write
+      tags:
+      - Placement Groups
+      summary: Placement Group Update
+      description: |
+        Change the `label` for a specific placement group. This is the only value you can update. However, you can [add](#placement-group-add-linode) more compute instances or [remove](#placement-group-remove-linode) existing ones.
+      operationId: updatePlacementGroup
+      x-linode-cli-action: update
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_write
+      requestBody:
+        description: >
+          Include the `label` parameter to update the name of your placement group.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PlacementGroupRequest/properties/label'
```

I was running into a small issue where the label was not being treated as a valid update field, here's the fix:
```suggestion
            schema:
              type: object
              properties:
                label:
                  $ref: '#/components/schemas/PlacementGroupRequest/properties/label'
```